tests/data/config/csc/_init.yaml

```diff
@@ -5,3 +5,4 @@ rules: []
 escalation: []
 escalation_url: ""
 escalation_timeout: 1
+narrative_server_url: https://tucson-teststand.lsst.codes/narrativelog
```

Comment:
Maybe use a fake url? Just to make sure people don't think we are hitting the TTS backend for testing.